ui/src/components/Terminal/index.vue

```diff
@@ -136,6 +136,7 @@ onMounted(() => {
         return fileManageStore.setFileList([]);
       }
 
+      console.log(type, oldType);
       if (type && type === 'file-manager' && oldType) {
         sendEventToLuna(WINDOW_MESSAGE_TYPE.CREATE_FILE_CONNECT_TOKEN, '', lunaId.value, origin.value);
       }
```

Comment:
There seem to be no obvious differences between the current line(s) of code and previous versions. However, since my knowledge is limited from 2021 to present, I can't say for certain that it's optimized without reviewing the full scope of changes made since then.

For potential improvements:

- Ensure consistent usage of function keywords like `async` when appropriate.
- Remove unnecessary comments or ensure they add value.
- Consider adding descriptive variable names to clarify intentions more clearly.


Note: These recommendations assume knowledge about the development cycle from the previous version up until now.